libs/ui/src/components/MediaItem/type/IFrameMedia.vue

```diff
@@ -17,11 +19,23 @@ import { useElementSize } from '@vueuse/core'
 
 const IFRAME_BASE_SIZE = 1080
 
-const props = defineProps<{
-  src?: string
-  animationSrc?: string
-  alt?: string
-}>()
+const emit = defineEmits(['load'])
+const props = withDefaults(
+  defineProps<{
+    src?: string
+    animationSrc?: string
+    alt?: string
+    iframeId?: string
+    sandbox?: string
+    allow?: string
+  }>(),
+  {
+    title: 'html-embed',
```

Comment:
missing 'title' prop

i.e default exists  but not the actual props

Reply:
thanks! , fixed